packages/libs/deer/README.md

```diff
@@ -0,0 +1,113 @@
+# deer
```

Comment:
Purposefully lower case?

Reply:
Yes, to keep in line with error-stack and how Rust crate naming usually is.

Reply:
I appreciate that the crate names are usually lower-case, but when referring to them in headings I haven't seen them left as lower-case before.

See:
- [Tokio](https://github.com/tokio-rs/tokio)
- Tokio [Tracing](https://github.com/tokio-rs/tracing)
- [Serde](https://github.com/serde-rs/serde)
- [Serde JSON](https://github.com/serde-rs/json)